doc/install.rst

```diff
@@ -28,24 +28,85 @@ There are different ways to install scikit-learn:
+          <span class="sk-expandable" data-packager="pip" data-os="linux" data-venv="">source sklearn-venv/bin/activate</span>
+          <span class="sk-expandable" data-packager="pip" data-os="mac" data-venv="">source sklearn-venv/bin/activate</span>
+          <span class="sk-expandable" data-packager="pip" data-os="windows" data-venv="">sklearn-venv\Scripts\activate</span>
+          <span class="sk-expandable" data-packager="pip" data-venv="">pip install -U scikit-learn</span>
+          <span class="sk-expandable" data-packager="pip" data-os="mac" data-venv="no">pip install -U scikit-learn</span>
+          <span class="sk-expandable" data-packager="pip" data-os="windows" data-venv="no">pip install -U scikit-learn</span>
+          <span class="sk-expandable" data-packager="pip" data-os="linux" data-venv="no">pip3 install -U scikit-learn</span>
+          <span class="sk-expandable" data-packager="conda" data-venv="">conda create -n sklearn-env</span>
+          <span class="sk-expandable" data-packager="conda" data-venv="">conda activate sklearn-env</span>
+          <span class="sk-expandable" data-packager="conda">conda install scikit-learn </span>
+       </code>
+       </pre></div>
+
+In order to check your installation you can use
+
+.. raw:: html
+
+   <div><pre>
+     <code>
+       <span class="sk-expandable" data-packager="pip" data-os="linux" data-venv="no">python3 -m pip show scikit-learn  # to see which version and where scikit-learn is installed</span>
+       <span class="sk-expandable" data-packager="pip" data-os="linux" data-venv="no">python3 -m pip freeze  # to see all packages installed in the active virtualenv</span>
+       <span class="sk-expandable" data-packager="pip" data-venv="">python -m pip show scikit-learn  # to see which version and where scikit-learn is installed</span>
+       <span class="sk-expandable" data-packager="pip" data-venv="">python -m pip freeze  # to see all packages installed in the active virtualenv</span>
+       <span class="sk-expandable" data-packager="pip" data-os="windows" data-venv="no">python -m pip show scikit-learn  # to see which version and where scikit-learn is installed</span>
+       <span class="sk-expandable" data-packager="pip" data-os="windows" data-venv="no">python -m pip freeze  # to see all packages installed in the active virtualenv</span>
+       <span class="sk-expandable" data-packager="pip" data-os="mac" data-venv="no">python -m pip show scikit-learn  # to see which version and where scikit-learn is installed</span>
+       <span class="sk-expandable" data-packager="pip" data-os="mac" data-venv="no">python -m pip freeze  # to see all packages installed in the active virtualenv</span>
+       <span class="sk-expandable" data-packager="conda">conda list scikit-learn  # to see which scikit-learn version is installed</span>
+       <span class="sk-expandable" data-packager="conda">conda list  # to see all packages installed in the active conda environment</span>
```

Comment:
One could also add the following command:

```
python -c "import sklearn; sklearn.show_versions()"
```

that would become the following under Linux without venv/conda env:

```
python3 -c "import sklearn; sklearn.show_versions()"
```

Reply:
Done in 16eff3a94c7c011211bb774ffdd854e2e568eeac

Reply:
Those instructions should stay displayed when the conda package manager is selected.